manifest_template.yml

```diff
@@ -3,29 +3,47 @@ PRODUCT_DISPLAY_NAME: Communications Manager
 DESCRIPTION: Use this API to send messages to citizens via email, SMS, the NHS App or letter.
 APIGEE_ENVIRONMENTS:
   - name: internal-dev
-    display_name: Internal Development
     has_mock_auth: true
-    app_ratelimit: '6000pm'
-    app_quota: '1200'
-    global_ratelimit: '18000pm'
-    global_quota: '6000'
+    variants:
+    - name: internal-dev
```

Comment:
Does this `name` field need declaring if it's in the level above? Is it overwriting what's in the level above?

Reply:
I notice you're now iterating over the variants in the manifest_template.xml, is it not possible to pull the name from the level above

Reply:
This name is used to build apigee products. We need 2 different products for ref, that's why I had to move it under variants.

Reply:
The level above is the environment name, the name under variant is part of the product name.

Reply:
Ahhh I see, happy to approve in that case